addons/product/views/product_views.xml

```diff
@@ -471,15 +471,9 @@
                                 <field name="currency_id" invisible="1"/>
                                 <field name="cost_currency_id" invisible="1"/>
                                 <field name="name" class="oe_inline" placeholder="e.g. Cheese Burger" string="Product Name"/>
-                                <label for="barcode" class="mt-1"/>
-                                <div name="barcode_autofill" class="d-flex">
-                                    <field name="barcode" class="oe_inline" placeholder="e.g. 1234567890"/>
-                                </div>
-                                <label for="list_price" class="mt-1"/>
-                                <div name="sales_price">
-                                    <field name="list_price" class="oe_inline" widget="monetary"
-                                            options="{'currency_field': 'currency_id', 'field_digits': True}"/>
-                                </div>
+                                <field name="barcode" class="oe_inline" placeholder="e.g. 1234567890" widget="productScanner"/>
```

Comment:
productScanner widget defined in `pos_barcode` module and you are using it here so it'll get traceback when `pos_barcode` module is not install :thinking: 
```suggestion
                                <field name="barcode" class="oe_inline" placeholder="e.g. 1234567890">
```
Add this attribute in `pos_barcode` module. No?

Reply:
Not in pos_barcodelookup but in main product_barcodelookup so whenever product is installed that will also be installed automatically but yes there can be a scenario if user manually uninstall the product_barcodelookup. so i have added it from product_barcodelookupplease check it

Reply:
Oh I didn't see your enterprise PR :see_no_evil: 
Yes also not everyone uses enterprise version :slightly_smiling_face: